services/appveyor/appveyor-tests.tester.js

```diff
@@ -3,19 +3,19 @@
 const Joi = require('joi')
 
 const isAppveyorTestTotals = Joi.string().regex(
-  /^(?:[0-9]+ (?:passed|skipped|failed)(?:, )?)+$/
+  /^[0-9]+ passed(, [0-9]+ failed)?( [0-9]+ skipped)?$/
```

Comment:
Isn't the comma separator missing for `skipped`?

Reply:
Indeed! Good spot!